concrete/single_pages/account/messages.php

```diff
@@ -244,12 +244,12 @@
 
 			<table class="table table-striped" border="0" cellspacing="0" cellpadding="0">
 			<tr>
-				<th class="ccm-profile-messages-item-name"><?=t('Mailbox')?></th>
```

Comment:
> I feel like this page is likely to be customized with CSS in a site and removing this might have consequences.

That's exactly for this reason that I added the new `ccm-profile-messages-item-mailbox` and kept the style of `ccm-profile-messages-item-name`: the mailbox name must not have a width of 100%, but the cells with the email subject needs that 100%